### DISCUSSION TITLE
大型 PR 的预期合并顺序

### DISCUSSION THREAD

**Discussion by LTCatt** [Meloong-Git/PCL#4234] [讨论]
基于 [B 站目前的投票结果](https://www.bilibili.com/read/cv36054339/)，预期从 2.8.4~5 开始按照如下顺序合并大型 PR：
#4020 (下载资源包) → #4206 (导出整合包) → #4214 (Quilt) → #4145 (多语言)
其他未合并的小型 PR 均预期在 2.8.3 合并。

![image](https://github.com/user-attachments/assets/b52ae6f8-0fdf-4655-bb4d-6e250fd9bd28)

**Comment by allMagicNB** [comment]
![Screenshot_20240712_132514_tv.danmaku.bili.jpg](https://github.com/user-attachments/assets/48132f4a-dc2d-4289-8948-d910a997238b)

**Reply by iPodtouch6557** [reply]
这个东西就是为了筛选将那些看不懂问题和不肯好好答题的人，使调查结果更准确
~所以你被筛掉了~

**Comment by allMagicNB** [comment]
~~所以 2.8.3 啥也没合并，对吗？~~

**Reply by Deep-Dark-Forest** [reply]
主要是有一个突发的严重 bug

**Reply by LTCatt** [reply]
这里说的版本号是指正常的更新预期，紧急的加更除外（
